Mathlib/Analysis/ConstantSpeed.lean

```diff
@@ -12,7 +12,7 @@ import Mathlib.Analysis.BoundedVariation
 # Constant speed
 
 This file defines the notion of constant (and unit) speed for a function `f : ℝ → E` with
-pseudo-emetric structure on `E` with respect to a set `s : Set ℝ` and "speed" `l : ℝ≥0`, and shows
+pseudo-metric structure on `E` with respect to a set `s : Set ℝ` and "speed" `l : ℝ≥0`, and shows
```

Comment:
This is correct in the original version.

Reply:
Ops, sorry. I'm going to correct it.

Reply:
@mcdoll Done.